service/src/main/java/uk/gov/hmcts/reform/fpl/service/TaskListRenderer.java

```diff
@@ -149,7 +150,11 @@ public String render(List<Task> allTasks) {
                 lines.add(renderLink(task) + renderImage("in-progress.png", "In progress"));
                 break;
             case COMPLETED:
-                lines.add(renderLink(task) + renderImage("information-added.png", "Information added"));
+                if (useFinishedTag(task)) {
```

Comment:
Since this method is private and very deep into the code, you can push the whole code into some other class, like TaskListCompletedMarkGenerator(), so that can be unit tested in isolation in TaskListCompletedMarkGeneratorTest instead and mock the component on TaskListRenderTest.